WalletWasabi/WabiSabi/Client/RoundStateUpdater.cs

```diff
@@ -4,18 +4,29 @@
 using System.Threading;
 using System.Threading.Tasks;
 using WalletWasabi.Bases;
+using WalletWasabi.Tor.Http;
+using WalletWasabi.Tor.Socks5.Pool.Circuits;
 using WalletWasabi.WabiSabi.Backend.PostRequests;
 using WalletWasabi.WabiSabi.Models;
+using WalletWasabi.WebClients.Wasabi;
 
 namespace WalletWasabi.WabiSabi.Client
 {
 	public class RoundStateUpdater : PeriodicRunner
 	{
+		private readonly PersonCircuit? _personCircuit;
```

Comment:
`RoundStateUpdater` shound't be aware or the existence of Tor circuites.

Reply:
I'll clean it up. Currently, I still don't understand the issue too much.